packages/xpetra/src/Vector/Xpetra_EpetraVector.cpp

```diff
@@ -128,8 +134,8 @@ template class EpetraVectorT<long long, Kokkos::Compat::KokkosOpenMPWrapperNode
 template Epetra_Vector & toEpetra<long long,Kokkos::Compat::KokkosOpenMPWrapperNode>(Vector<double, int, long long, Kokkos::Compat::KokkosOpenMPWrapperNode> &);
 template const Epetra_Vector & toEpetra<long long,Kokkos::Compat::KokkosOpenMPWrapperNode>(const Vector<double, int, long long, Kokkos::Compat::KokkosOpenMPWrapperNode> &);
 #endif
-#ifdef HAVE_TPETRA_INST_CUDA
-typedef Kokkos::Compat::KokkosCudaWrapperNode default_node_type;
+#ifdef HAVE_TPETRA_INST_HIP
```

Comment:
Why was the `HAVE_TPETRA_INST_CUDA` clause removed?

Reply:
Fixed